packages/libs/lambda-at-edge/src/build.ts

```diff
@@ -321,6 +321,14 @@ class Builder {
           }
         }
       ),
+      // copy chunks if present and not using serverless trace
```

Comment:
Seems the build produces chunks even with serverless trace target. Are you sure they can be left out? Or are they added elsewhere?

Reply:
Never mind, they get added with the other traced files.